src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -131,59 +117,51 @@
     short-summary: If given, lists the children in all levels of hierarchy.
 examples:
   - name: Get a management group.
-    text: >
-        az account management-group show --name GroupName
+    text: az account management-group show --name MyGroupName
   - name: Get a management group with children in the first level of hierarchy.
-    text: >
-        az account management-group show --name GroupName -e
+    text: az account management-group show --name MyGroupName --expand
   - name: Get a management group with children in all levels of hierarchy.
-    text: >
-        az account management-group show --name GroupName -e -r
+    text: az account management-group show --name MyGroupName --expand --recurse
 """
 
 helps['account management-group subscription'] = """
 type: group
-short-summary: Subscription operations for Management Groups.
+short-summary: Add or remove subscriptions in a management group.
```

Comment:
In fact, because it is uncertain whether other operations will be expanded in the future, it is recommended to use a unified and broad description `Manage`
```suggestion
short-summary: Manage subscriptions in a management group.
```